### DISCUSSION TITLE
About GitHub Badges

### DISCUSSION THREAD

**Discussion by vishnubansal001** [community/community#48958] [New to GitHub] [answered]
How to take GitHub YOLO Badge on our id ?
Can Anyone elaborate the steps to take this badge?.

**Comment by bbkx226** [comment, marked as answer]
Merge the branches of your repo without reviewing it, and you'll then get the badge and shown in your profile

**Reply by vishnubansal001** [reply]
How to merge without reviewing it ?? 
Can explain me in steps pls like from starting with creating a new repo demo and then a new branch of it ....
Please

**Reply by bbkx226** [reply]
Alright, let me explain it in brief:
1. Create a pull request on your remote repository to indicate the changes you've made.
2. Open the pull request to allow others to review your code.
3. Add a reviewer to your repository by going to the top right corner and selecting "Add reviewer".
4. Before the reviewer can merge your code, you need to merge your changes into the remote repository.
